### PR TITLE
geo-coding with ActiveJob/SuckerPunch

### PR DESCRIPTION
**Pull request by phoet**
@kikito i added an example for how we could use ActiveJob with SuckerPunch

### CODE REVIEW

File: app/extensions/active_job/core.rb [hunk 1]
```diff
@@ -0,0 +1,113 @@
+# https://github.com/rails/rails/blob/65542e27979fce1f6b0893a00e4d51849616de34/activejob/lib/active_job/core.rb
+module ActiveJob
```
Comment (phoet) [inline app/extensions/active_job/core.rb]: had to get latest head to support full custom deserialization support

**Comment by phoet** [comment]
@kikito i don't know if you have seen this PR? in case we want to continue down the path of having a background job fetch the previews

**Comment by kikito** [comment]
Yes, I have seen it and read the code. I have not had time to test it on my machine though. Does it work like we want? Have you tried it in heroku?

**Comment by phoet** [comment]
Yes, it works, was a bit of a hassle to set up for testing, heroku was
painless.
Am 09.02.2015 20:45 schrieb "Enrique García" <notifications@github.com>:

**Comment by kikito** [comment]
In that case, why isn't this merged? Are you waiting on me to provide a preview "job", or is there something else missing?

**Comment by phoet** [comment]
i was wondering if this additional complexity is even necessary given the scheduler that is running every hour to trigger building the previews. if that is sufficient for now, i'll just close this one.

**Comment by kikito** [comment]
Oh I didn't see that scheduler. Where is that?

**Comment by phoet** [comment]
@kikito yes exactly that. for now it runs every hour and i get notified about exceptions etc. the only thing that won't be supported right now is updating the resource-url. that would still point to the old preview.

**Comment by kikito** [comment]
Yeah, that makes me think that the preview-type and preview-code should be editable by admins - so at least they can clear them out to trigger a preview refresh. Otherwise once you get a preview, you are stuck with it for ever.

**Comment by phoet** [comment]
@kikito i think it would be simple to have another button saying "refresh_preview" in the admin? anyways, we can find something simple.

**Pull request closed without merging**